deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -115,7 +152,7 @@ data:
     </source>
     @include logs.source.containers.conf
     @include logs.source.systemd.conf
-  logs.output.conf: |
+  logs.output.conf: |+
```

Comment:
should `|+` be `|-` ?

Reply:
yeah, even I noticed that. Upon searching found that its because of an extra trailing blank line. ``` |+: "keep": keep the line feed, keep trailing blank lines.```
Will remove that.